buidl/test/test_network.py

```diff
@@ -94,40 +94,40 @@ def test_serialize(self):
         self.assertEqual(get_data.serialize().hex(), hex_msg)
 
 
-# FIXME: make this work
```

Comment:
Wohoo, love that this got fixed!

Close this issue now?
https://github.com/buidl-bitcoin/buidl-python/issues/27